_opcodes/op453-iwd2.html

```diff
@@ -10,4 +10,10 @@
 iwd2: 1
 pst: 0
 ---
-Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages. This effect is used internally.
+Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages.
+
+{% capture note %}
+This effect is used internally.
```

Comment:
Many are, not noteworthy enough. It just has a connotation that it's use might be different or difficult. Same for the other feat spells.